### PR TITLE
replace stella plugin

### PR DESCRIPTION
**Pull request by carlomazzaferro**
## Type of change

- [ ] Docs change / dependency upgrade
- [ ] Configuration / tooling changes
- [ ] Refactoring
- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Requires changes in customer code

## Related Issue(s)

Fixes

### CODE REVIEW

File: packages/sdk/tasks/one-time/stella-upgrade.ts [hunk 1]
```diff
@@ -0,0 +1,89 @@
...
+    ];
+    const newImplementations = [
+      sdk.chainDeployment.CErc20Delegate.address,
+      sdk.chainDeployment.CErc20PluginDelegate.address,
+      sdk.chainDeployment.CErc20PluginRewardsDelegate.address,
+    ];
+    const arrayOfFalse = [false, false, false];
+    const arrayOfTrue = [true, true, true];
+
+    if (sdk.chainDeployment.CErc20PluginRewardsDelegate.address !== "0x32Be4b977BaB44e9146Bb414c18911e652C56568") {
```
Comment (vminkov) [inline packages/sdk/tasks/one-time/stella-upgrade.ts:46]: no, this is the address of the market delegator/proxy - the correct address of the current `CErc20PluginRewardsDelegate ` is `0xad0538f54d7a503FE138b9fB62711B2034E0FD94 `

File: packages/sdk/tasks/one-time/stella-upgrade.ts [hunk 2]
```diff
@@ -0,0 +1,89 @@
...
+      args: [],
+      log: true,
+      waitConfirmations: 1,
+    });
+    console.log("CErc20PluginRewardsDelegate: ", erc20PluginRewardsDel.address);
+
+    const erc20PluginRewardsDelegate = await ethers.getContract("CErc20PluginRewardsDelegate", signer);
+
+    const oldImplementations = [
+      ethers.constants.AddressZero,
```
Comment (vminkov) [inline packages/sdk/tasks/one-time/stella-upgrade.ts:34]: the first 3 values of the `oldImplementations` and `newImplementations` arrays are irrelevant and can be removed

File: packages/sdk/tasks/one-time/stella-upgrade.ts [hunk 3]
```diff
@@ -0,0 +1,89 @@
...
+      arrayOfFalse,
+      arrayOfTrue
+    );
+
+    await tx.wait();
+    console.log("_editCErc20DelegateWhitelist:", tx.hash);
+
+    const abiCoder = new ethers.utils.AbiCoder();
+
+    const pluginAddress = ethers.constants.AddressZero;
```
Comment (vminkov) [inline packages/sdk/tasks/one-time/stella-upgrade.ts:68]: better use the same current plugin address for `pluginAddress`

**Review by vminkov — CHANGES_REQUESTED** [review]

File: packages/sdk/tasks/one-time/stella-upgrade.ts (hunk 1 above)
Comment (carlomazzaferro) [reply packages/sdk/tasks/one-time/stella-upgrade.ts:46]: Good catch

**Pull request merged**